styles/global-css-settings.scss

```diff
@@ -0,0 +1,73 @@
+$box-shadow-small: 0 4px 8px -1px rgba(0, 0, 0, 0.2);
+$box-shadow-medium: 0px 6px 20px -2px rgba(0, 0, 0, 0.2);
+$box-shadow-large: 0px 15px 50px -10px rgba(0, 0, 0, 0.3);
+
+$border-radius-extra-small: 2px;
+$border-radius-small: 4px;
+$border-radius-medium: 8px;
+$border-radius-big: 16px;
+
+$spacing-xxs: 2px;
+$spacing-xs: 4px;
+$spacing-xs-small: 6px;
+$spacing-small: 8px;
+$spacing-small-medium: 12px;
+$spacing-medium: 16px;
+$spacing-large: 24px;
+$spacing-xl: 32px;
+$spacing-xxl: 48px;
+$spacing-xxxl: 64px;
+
```

Comment:
these are spacing tokens that are not only for the documentation but the actual design tokens.
In that case, I think we should take them directly from the src instead of duplicating